plugins/utils/nvim-osc52.nix

```diff
@@ -8,38 +8,44 @@
 with lib;
 {
   options.plugins.nvim-osc52 = {
-    enable = mkEnableOption "nvim-osc52, a plugin to use OSC52 sequences to copy/paste";
+    enable = mkEnableOption "nvim-osc52, a plugin to use OSC52 sequences to copy/paste.";
```

Comment:
mkEnableOption adds its own period. This will render as "Whether to enable nvim-osc52, a plugin to use OSC52 sequences to copy/paste.."

Reply:
Thanks for the pointer! Fix'd.